.github/workflows/ci.yml

```
@@ -280,7 +280,7 @@ jobs:
         uses: grafana/plugin-ci-workflows/actions/plugins/setup@main
         with:
           go-version: ${{ inputs.go-version || env.DEFAULT_GO_VERSION }}
-          node-version: ${{ inputs.node-version || env.DEFAULT_NODE_VERSION }}
+          node-version: ${{ inputs.node-version || (hashFiles(format('{0}/.nvmrc', inputs.plugin-directory)) == '' && env.DEFAULT_NODE_VERSION || '') }}
```

Nit: I'd add a comment above to specify the order of the values, something like:

```suggestion
          // Priority: 1. inputs.node-version, 2. .nvmrc file, 3. workflow-level DEFAULT_NODE_VERSION
          node-version: ${{ inputs.node-version || (hashFiles(format('{0}/.nvmrc', inputs.plugin-directory)) == '' && env.DEFAULT_NODE_VERSION || '') }}
```